lib/cereal/utils.ex

```diff
@@ -38,12 +39,52 @@ defmodule Cereal.Utils do
 
   defp normalize_relationship_path([]), do: []
   defp normalize_relationship_path([rel | rest]) do
-    Keyword.put([], String.to_atom(rel), normalize_relationship_path(rest))
+    case string_to_atom(rel) do
+      nil -> []
+      key -> Keyword.put([], key, normalize_relationship_path(rest))
+    end
   end
 
   defp deep_merge_relationship_paths(left, right), do: Keyword.merge(left, right, &deep_merge_relationship_paths/3)
   defp deep_merge_relationship_paths(_, left, right), do: deep_merge_relationship_paths(left, right)
 
+  @doc """
+  Takes a keyword list of comma separated strings keyed by serializer name and
+  converts the strings into lists of atoms. Implicitly removes any binary string
+  that cannot be safely converted to an atom.
```

Comment:
that cannot be safely converted to an _existing_ atom?

Reply:
There's at least one other case where a binary-to-atom conversion will fail: when the binary is too long. Since there might be other unknown cases I figured I'd leave it generic indicating that there are at least a couple cases where conversion would fail.